wfsim/strax_interface.py

```diff
@@ -317,11 +288,28 @@ def __call__(self, instructions, channels, timings):
                  help="Terminate processing if any one mailbox receives "
                       "no result for more than this many seconds"),
     strax.Option('fax_config',
-                 default='fax_config_nt.json'),
+                 default='https://raw.githubusercontent.com/XENONnT/private_nt_aux_files/master/sim_files/fax_config_nt.json?token=AHCU5AZMPZABYSGVRLDACR3ABAZUA'),
     strax.Option('gain_model',
-                 default=('to_pe_per_run', 'to_pe_nt.npy'),
+                 default=('to_pe_per_run', 'https://github.com/XENONnT/private_nt_aux_files/blob/master/sim_files/to_pe_nt.npy?raw=true'),
                  help='PMT gain model. Specify as (model_type, model_config)'),
-    strax.Option('detector', default='XENONnT', track=True),)
+    strax.Option('detector', default='XENONnT', track=True),
+    strax.Option('channel_map', track=False, type=immutabledict,
+                 help="immutabledict mapping subdetector to (min, max) "
+                      "channel number. Provided by context"),
+    strax.Option('n_tpc_pmts', track=False,
+<<<<<<< HEAD
```

Comment:
I tried to run this branch but I get an error for this line when importing wfsim.